### DISCUSSION TITLE
Notifications fails with any kind of network latency

### DISCUSSION THREAD

**Discussion by peppy** [community/community#87630] [Mobile]
### Select Topic Area

Bug

### Body

There are two or more failing cases when triaging notifications on a network with latency. I would suggest testing with induced latency.

- Swiping gets reset halfway through, likely due to previous swipe’s response arriving 

https://github.com/community/community/assets/191335/ef89146f-ad53-4e84-9199-e74490b36e22

- Sometimes actions get undone. When swiping three or four notifications away, one will return at random (haven’t caught on video yet)

**Comment by eliperkins** [comment]
Hey @peppy, thanks for this bug report! This is definitely not intended behavior, and I can see how frustrating it would be to take action on multiple notifications.

I've created an issue for this bug on our internal tracking repo. I'll work with our engineering team to get this addressed ASAP, and follow back up with you when it's been fixed. Thanks for helping us improve GitHub Mobile. 🙇
